docs/Build-for-RPi2.md

```diff
@@ -3,10 +3,11 @@
 ### Prerequisite
 #### Linux
 
-Install arm linux cross compiler.
+Install arm linux cross compiler and i2c library.
 
 ``` bash
-sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf
+sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf i2c-tools libi2c-dev
+sudo mv /usr/arm-linux-gnueabihf/include/linux/i2c-dev.h /usr/arm-linux-gnueabihf/include/linux/i2c-dev.h.kernel
```

Comment:
It's include path setup problem, but If we want to fix it completely, we need to copy include folder from raspberry pi file system to ubuntu and add option 'sysroot' when cross-compile. (maybe original 'i2c-dev.h' file in raspberry pi is renamed to 'i2c-dev.h.kernel' and new '
i2c-dev.h' is generated, so it will work fine) It will work with MacOS using this approach.
However, it is not possible to cross-compile until copying them from Raspberry Pi.

This approach is too complex, so I found another approach. It is simply rename header file in arm include folder, so cross-compiler will use header file in ubuntu file system. I also think it's not perfect approach, but it works.